### ISSUE TITLE
Opening and exiting menu several times causes freeze

### ISSUE THREAD

**Issue by amitkulk123** [DS-Homebrew/nds-bootstrap#1581]
New Nintendo 3DS LL 

Launched via SD. I region switched to US and am running software firmware 11.17.0-50U. 

**Version used:**
nds-bootstrap release v0.72.0

**Game tested/used/applicable:**
This happens with every game I've tested. I've so far tested it with Pokemon Platinum, Pokemon Diamond, and osu tatakae ouendan and it's happened with every game. 

**Save file:**
This doesn't effect a specific game but I'll upload the files if they're really needed. 

**Expected Behavior:**
It shouldn't crash when loading the menu. 

**Actual Behavior:**
It crashes whenever I open the menu several times or right before I put it to sleep. I can't find a way to unfreeze except by quitting and returning to the 3DS home menu. 

### Steps to reproduce
1. Open any game 
2. Open the menu with L+R+Down+Select
3. Either close the lid while the menu is open to put the console to sleep or open/close the menu several times. It'll eventually freeze or black screen.

### nds-bootstrap.ini
```
[NDS-BOOTSTRAP]
NDS_PATH = sd:/roms/nds/Pokemon - Platinum Version (USA) (Rev 1).nds
SAV_PATH = sd:/roms/nds/saves/Pokemon - Platinum Version (USA) (Rev 1).sav
BOOST_CPU = -1
DEBUG = 0
LOCK_ARM9_SCFG_EXT = 0
LOGGING = 1
B4DS_MODE = 0
ROMREAD_LED = 0
DMA_ROMREAD_LED = -1
PRECISE_VOLUME_CONTROL = 0
SDNAND = 0
MACRO_MODE = 0
SLEEP_MODE = 1
SOUND_FREQ = 0
CONSOLE_MODEL = 2
HOTKEY = 284
CACHE_FAT_TABLE = 0
BOOST_VRAM = 0
CARD_READ_DMA = -1
ASYNC_CARD_READ = -1
DSI_MODE = 1
SWI_HALT_HOOK = 1
EXTENDED_MEMORY = 0
DONOR_SDK_VER = 0
PATCH_MPU_REGION = 0
PATCH_MPU_SIZE = 0
LANGUAGE = -1
REGION = -2
RAM_DRIVE_PATH = sd:/null.img
GUI_LANGUAGE = en
USE_ROM_REGION = 1
FORCE_SLEEP_PATCH = 0
AP_FIX_PATH =
```

### Logs
I tried enabling logging by setting it to 1 but it doesn't create the file for some reason

### Other notes
Recorded a couple of videos of the issue:
Entering in and out of menu several times:
![ezgif-2-1b003d62c9](https://github.com/DS-Homebrew/nds-bootstrap/assets/9218849/f4afb213-a4c8-4d20-936d-c97a766b45ce)
Entering menu before sleep:
![ezgif-2-5d46b3ad84](https://github.com/DS-Homebrew/nds-bootstrap/assets/9218849/fba7f8c9-3716-4192-9c8e-072a8501d607)

### Will you be available for future issues and/or replies?
Yes

**Comment by RocketRobz** [comment]
Black screen issue has been fixed by e2922638

**Comment by amitkulk123** [comment]
> Black screen issue has been fixed by [e292263](https://github.com/DS-Homebrew/nds-bootstrap/commit/e2922638544fb1e8a851b6d6ee4e423f5492fbaa)

@RocketRobz I downloaded the latest nightly release (it shows up as nds-bootstrap-bda8d01) and looks like the issue with when it black screens/freezes on wake from sleep is fixed!

However, it still freezes when opening and closing the menu several times. The number of times varies but it eventually freezes. 

![ezgif-1-f24d9658f5](https://github.com/DS-Homebrew/nds-bootstrap/assets/9218849/c035fa61-cfca-4e5d-b6b6-a4b45bc00e18)

**Comment by lesar8** [comment]
The same problem happened to me as well. I have a DSI XL firmware 1.43 JP, running  unlaunch and Twilight Menu v26.5.0 via SD. Opening and exiting menu several times (usually 7-8 times) will causes freeze for most (if not all) games.
